.changeset/swift-colts-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@svenvw/fdm-core": minor
+---
+
+Add `b_lu_n_harvestable` as property of cultivation catalogue. It represents the nitrogen content of the harvestable parts of the crop for a cultivation and can be used as default
```

Comment:
add unit please

Reply:
Will be resolved in 572cd350cd8e2a61ad469f71212b7fa19d48f5b1